runtime/doc/news.txt

```diff
@@ -95,6 +95,7 @@ CHANGED FEATURES                                                 *news-changes*
 The following changes to existing APIs or features add new behavior.
 
 • 'exrc' is no longer marked deprecated.
+• 'exrc' now supports `.nvimrc.lua` file.
```

Comment:
Changes should be listed in reverse chronological order (so put this bullet on top).

Reply:
Hmm... why? Grouping things is useful, and avoids merge conflicts as a bonus.

Reply:
They will still be grouped, I meant to put the new bullet at line 96 rather than line 98.

>Hmm... why?

Partially because this is how new items have been added thus far, so we want to maintain ordering (assuming we care about ordering, which I think we do). For someone who checks news.txt recently for new items, it's easier to see new items at the top of each list rather than appended to the bottom (though perhaps that's subjective -- I find it more noticeable at least).

Reply:
If `news.txt` is to be in reverse chronological order, the exception could be related follow-up PRs. In that case grouping might be more valuable than keeping it chronologically ordered? I think it would be if you see it as a release list but it would be less useful for someone on a dev build.

Reply:
This may need a broader discussion, because I certainly see benefits to both.

* For people following HEAD, listing changes in chronological order (reverse or otherwise) is useful to see what has changed _most recently_.
* For people only updating to stable releases, chronology doesn't matter, and grouping items by "theme" is probably more useful.

https://github.com/neovim/neovim/issues/21431 might be a way out of this. For now, I'd still recommend sticking to reverse chronological order as that is the order that every other item in news.txt has been added thus far. If we change it later to be grouped by "theme", we can change everything all at once.

Reply:
Sounds good. Can revisit later.